python/aqora_cli/qiskit/client.py

```
@@ -0,0 +1,290 @@
+        return json.loads(json.dumps(dict(options)))
+    except TypeError as exc:
+        raise TypeError("Backend options must be JSON-serializable") from exc
+
+
+def _put_json_payload(upload_url: str, payload: str) -> str:
+    request = urllib.request.Request(
+        upload_url,
+        data=payload.encode("utf-8"),
+        method="PUT",
+        headers={"Content-Type": "application/json"},
+    )
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)